src/coreclr/jit/emitxarch.cpp

```diff
@@ -13250,6 +13257,11 @@ BYTE* emitter::emitOutputLJ(insGroup* ig, BYTE* dst, instrDesc* i)
             break;
     }
 
+    if (jmp && id->idjIsJmpAlways == 1 && id->idCodeSize() == 0)
```

Comment:
For consistency with similar checks around `idjKeepLong`, remove the extra `= 1` here and at other places.
```suggestion
    if (jmp && id->idjIsJmpAlways && id->idCodeSize() == 0)
```